.github/workflows/check.yml

```diff
@@ -20,6 +20,14 @@ jobs:
 
       - name: mount bazel disk cache
         uses: actions/cache@v3
+        if: github.ref == 'refs/heads/master'
+        with:
+          path: "~/.cache/bazel_disk_cache"
+          key: bazel-disk-test-${{ matrix.toolchain }}-${{ matrix.feature }}
```

Comment:
this will fail to save the cache once it is established

is it work deleting the cache so that it can be updated when commits are merged?

Reply:
Do you think there's something low effort that I should do instead?
Next step I think would be to set up buildbuddy, but I don't want to spend the time to do that just yet.

Reply:
setting up buildbuddy was very low effort.

< 15 mins would be my guess

Reply:
OK, done. That was easier than expected!